### ISSUE TITLE
cryptography / pyopenssl / pip incompatibility    

### ISSUE THREAD

**Issue by bemoody** [MIT-LCP/physionet-build#1915]
The combination of these packages breaks pip (see pull #1911):
```
$ rm -rf ve1
$ virtualenv ve1
$ ./ve1/bin/pip install --no-deps pyopenssl==19.1.0 \
                                  cffi==1.15.1 \
                                  cryptography==39.0.1
$ ./ve1/bin/pip freeze
[...]
AttributeError: module 'lib' has no attribute 'X509_V_FLAG_CB_ISSUER_CHECK'
```
Omit any one of the three and it works.

**Comment by tompollard** [comment]
Thanks for getting to the bottom of this! First time I've seen something like this happen/

**Comment by tompollard** [comment]
@bemoody this remains an issue, but not for us. Can we close or is there a benefit in keeping it open?

**Comment by bemoody** [comment]
Yep, this was addressed by #1953.
